care/facility/models/tests/test_patient.py

```diff
@@ -23,3 +26,10 @@ def test_disease_state_recovery_is_aliased_to_recovered(self):
         patient.refresh_from_db()
 
         self.assertEqual(patient.disease_status, DiseaseStatusEnum.RECOVERED.value)
+
+    def test_age_validation(self):
+        patient = self.patient
+        patient.date_of_birth = now().today() + timedelta(days=365)
+        with self.assertRaises(ValidationError):
+            patient.full_clean()
+            patient.save(update_fields=["age","date_of_birth","year_of_birth"])
```

Comment:
the serialization validations wont be applied here, use the api test to try to create a new patient

Reply:
Ok I'll update them